app/views/exam_templates/_assign_errors.js.erb

```diff
@@ -0,0 +1,69 @@
+<script>
+  function rotate() {
+      PDFView.rotatePages(180);
+  }
+  PDFView.onLoadComplete = function() {
```

Comment:
Can you try to remove this part and see if the pages still load properly? `PdfAnnotationManager` is a class that's used for grading, but not here.